src/entity/Action.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+export enum ActionType {
+  Notification = 'notification',
+}
+
+@Entity()
+export class Action {
+  @Index()
+  @PrimaryColumn({ type: 'text' })
+  userId: string;
+
+  @Index()
```

Comment:
I don't think we need index on this field. We're not gonna search over it I guess

Reply:
Hmm. Good point, we may search at very rare cases but not very often. Will remove this one.